core/services/relay/evm/config_poller.go

```diff
@@ -76,7 +75,9 @@ func NewConfigPoller(lggr logger.Logger, cfg CPConfig) (evmRelayTypes.ConfigPoll
 }
 
 func newConfigPoller(lggr logger.Logger, client client.Client, destChainPoller logpoller.LogPoller, aggregatorContractAddr common.Address, configStoreAddr *common.Address, ld LogDecoder) (*configPoller, error) {
-	err := destChainPoller.RegisterFilter(logpoller.Filter{Name: configPollerFilterName(aggregatorContractAddr), EventSigs: []common.Hash{ld.EventSig()}, Addresses: []common.Address{aggregatorContractAddr}})
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Are we tracking these ones for follow up?

Reply:
Propagated this one until chainlink-common, then added a single additional TODO for this ticket: https://smartcontract-it.atlassian.net/browse/BCF-2887